app/dashboard-user/dashboard-user.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NativeScriptModule } from "nativescript-angular/platform";
+import { NativeScriptFormsModule } from "nativescript-angular/forms";
+import { NgModule } from "@angular/core";
+import { dashboardRouting } from "./dashboard-user.routing";
+import { DashboardUserComponent } from "./dashboard-user.component";
+import { BaseViewModule } from "../base-view";
+import { TileComponent } from "./tile/tile.component";
+import { ChartDirectiveWrapper } from "../chart-directive-wrapper.module";
+
+@NgModule({
+  imports: [
+    BaseViewModule,
+    ChartDirectiveWrapper,
```

Comment:
nativescript tells me i have 2 imports of the CHART_DIRECTIVES if i import it in the NgModules that actually need them... (stacktrace) They say i need to import it in a higher level component (like app.module) OR wrap it in a module and then import that instead...  Now the import in app.modules didnt work... But this did sooo..